crates/turbopack-css/src/chunk/writer.rs

```diff
@@ -36,6 +36,18 @@ pub async fn expand_imports(
                     close,
                 ));
             }
+            Some(CssImport::Composes(composed_chunk_item)) => {
+                let id = &*composed_chunk_item.to_string().await?;
+                writeln!(code, "/* composes({}) */", id)?;
```

Comment:
I'm not sure whether this should actually happen or not. @sokra ?

Reply:
I think a header is useful to separate the chunk items in the chunk.

Reply:
I don't think I saw this comment at all in front, which makes me think this doesn't run at all. I'll check again.